acknowlegements.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Acknowledgements
+---
+
+We would like to thank the following for their generous contributions to this
+site:
+
+* [@sukima](https://tritarget.org/)
```

Comment:
_pats himself on the back, eh?_

Reply:
Well, TBH a name there looked better then

```markdown
* TODO: Put something here.
```

But yeah. shameless self promotion. :blush:

Reply:
The Acknowledgement page is optional and in this simple case is likely overkill. With the possibility of PR #101 this also becomes redundant and more maintenance overhead. I think I should reevaluate this PR.